aioesphomeapi/model.py

```diff
@@ -245,6 +255,15 @@ def _convert_climate_swing_modes(value):
     return [ClimateSwingMode(val) for val in value]
 
 
+def _convert_climate_presets(value):
+    return [ClimatePreset(val) for val in value]
+
+
+def _convert_string(value):
+    # need to convert protobuf iterable to list, otherwise HA fails to serialize it as JSON
+    # pylint: disable=R1721
+    return [val for val in value]
```

Comment:
```suggestion
    return list(value)
```

Reply:
@Lumpusz Have you tried this suggestion?

Reply:
I have not tried this, did not think it was important. This is due to the fact that lint thinks it is already a list, but it is some kind of proto specific class, and without this HA fails.